addons/web/static/tests/views/fields/char_field_tests.js

```diff
@@ -182,9 +182,7 @@ QUnit.module("Fields", (hooks) => {
         );
```

Comment:
in the or in some?
the solution to what? you present your commit as a REF. Is it an adaptation required because of some other fix? If yes it would be nice to put the commit before that fix or merge it with it